docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -1,213 +1,184 @@
 # BigDL-LLM Installation: GPU
 
+## Windows
 
-## PyTorch 2.1
-### Prerequisite
+### Prerequisites
 
-```eval_rst
-.. tabs::
-   .. tab:: Linux
-
-      BigDL-LLM for GPU supports on Linux with PyTorch 2.1 has been verified on:
+BigDL-LLM on Windows supports Intel iGPU and dGPU.
```

Comment:
make it clear only PyTorch 2.1 is supported on windows

Reply:
add it